frontend/test/metabase-visual/static-visualizations/static-visualizations.cy.spec.js

```diff
@@ -0,0 +1,105 @@
+import { restore, setupSMTP, cypressWaitAll } from "__support__/e2e/cypress";
+import { USERS } from "__support__/e2e/cypress_data";
+import { SAMPLE_DATASET } from "__support__/e2e/cypress_sample_dataset";
+
+const { ORDERS_ID, ORDERS, PRODUCTS } = SAMPLE_DATASET;
+
+const { admin } = USERS;
+
+const visualizationTypes = ["line", "area", "bar", "combo"];
+
+const SENDING_EMAIL_TIMEOUT = 30000;
```

Comment:
This seems hacky/arbitrary.
Where did 30s come from? Empirically, or did you just want to make sure test is not failing so you bumped it up significantly?

Reply:
@nemanjaglumac empirically, because sometimes rendering static viz takes a while and I had timeouts on CI